features/networking/multus.feature

```diff
@@ -1272,3 +1272,48 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | route         |
     Then the output should contain:
       | 192.168.10.0                 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-29742
+  @admin
+  Scenario: Log pod IP and pod UUID when pod start	
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-macvlan.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>      |    
+      | ["spec"]["config"]        |'{ "cniVersion": "0.3.0", "type": "macvlan","mode": "bridge", "ipam": { "type": "whereabouts", "range": "192.168.22.100/24"} }' |
+    Then the step should succeed
+    
+    # Create a pod absorbing above net-attach-def
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
+      | ["metadata"]["name"]                                       | macvlan-bridge-whereabouts-pod1  |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | macvlan-bridge-whereabouts       |
+      | ["spec"]["containers"][0]["name"]                          | macvlan-bridge-whereabouts       |
+      | ["spec"]["nodeName"]                                       | <%= cb.nodes[0].name %>          |
+    Then the step should succeed
+    And the pod named "macvlan-bridge-whereabouts-pod1" becomes ready
+    And evaluation of `pod.ip` is stored in the :pod_eth0_ip clipboard
+
+    When I run the :get admin command with:
+      | resource      | pod                                |
+      | n             | <%= project.name %>                |
+      | o             | jsonpath={.items[*].metadata.uid}  |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :pod_uid clipboard
+   
+    And I execute on the "macvlan-bridge-whereabouts-pod1" pod:
+      | ifconfig | net1 |
```

Comment:
`ifconfig` has been deprecated since 2009

https://serverfault.com/questions/633087/where-is-the-statement-of-deprecation-of-ifconfig-on-linux

`ip` has `-4` and `-brief`

```
# ip -4  -brief a  show
lo               UNKNOWN        127.0.0.1/8
eth0@if19        UP             10.129.2.6/23
```

Centos iproute2 is old, but the newer versions have JSON output with `-json` so it is parseable.